core/lib/protobuf_config/src/proto/core/consensus.proto

```diff
@@ -87,6 +87,9 @@ message Config {
   // Maximal allowed size of the sync batches.
   optional uint64 max_batch_size = 10; // required; bytes
 
+  // View timeout for the consensus protocol.
+  optional uint64 view_timeout = 13; // required; milliseconds
```

Comment:
std.Duration

Reply:
I did try it but then got into an issue where Rust couldn't convert between proto::std::Duration and time::Duration and ended up going back. I'll give it a try again.